src/schema/graphql.schema.ts

```diff
@@ -240,6 +240,7 @@ export interface User {
     firstName: string;
     middleName?: string;
     lastName: string;
+    status: string;
```

Comment:
Can we make use of [Graphql Enum](https://graphql.org/learn/schema/#enumeration-types) to ensure this is a well defined value and not some random string?